cooler_manipulation.ipynb

```diff
@@ -0,0 +1,1524 @@
+{
```

Comment:
<p>should this be done earlkier? If not, maybe add one sentence to explain that cooler is built with hdf5 storage, so direct manipulations can be done with the python library h5py</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>